tests/fuzz/s2n_server_finished_recv_test.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/* Target Functions: s2n_server_finished_recv */
+
+#include <stdint.h>
+
+#include <openssl/crypto.h>
+#include <openssl/err.h>
+
+#include "api/s2n.h"
+#include "stuffer/s2n_stuffer.h"
+#include "tls/s2n_connection.h"
+#include "tls/s2n_tls.h"
+#include "utils/s2n_safety.h"
+#include "s2n_test.h"
+
+static const uint8_t TLS_VERSIONS[] = {S2N_TLS10, S2N_TLS11, S2N_TLS12, S2N_TLS13, S2N_SSLv3};
+
+int s2n_fuzz_test(const uint8_t *buf, size_t len)
+{
+    /* We need at least one byte of input to set parameters */
+    S2N_FUZZ_ENSURE_MIN_LEN(len, 1);
+
+    /* Setup */
+    struct s2n_connection *client_conn = s2n_connection_new(S2N_SERVER);
+    notnull_check(client_conn);
+    GUARD(s2n_stuffer_write_bytes(&client_conn->handshake.io, buf, len));
+
+    /* Pull a byte off the libfuzzer input and use it to set parameters */
+    uint8_t randval = 0;
+    GUARD(s2n_stuffer_read_uint8(&client_conn->handshake.io, &randval));
+    client_conn->actual_protocol_version = TLS_VERSIONS[randval % s2n_array_len(TLS_VERSIONS)];
+
+    uint8_t finished_len = S2N_TLS_FINISHED_LEN;
+    if (TLS_VERSIONS[randval % s2n_array_len(TLS_VERSIONS)] == S2N_SSLv3) {
```

Comment:
why not just calculate TLS_VERSIONS[randval % s2n_array_len(TLS_VERSIONS)] once? 
This is more of a readability/editability concern than anything else.

Reply:
Since other fuzz tests don't pre-calculate it (due to not needing to), I thought that it would be more readable to do it this way so that it doesn't look like a new and different process is taking place for this fuzz tests.